cross/libvpx/Makefile

```diff
@@ -28,40 +30,33 @@ include ../../mk/spksrc.archs.mk
 
 ifeq ($(findstring $(ARCH),$(ARMv5_ARCHS)),$(ARCH))
 CONFIGURE_ARGS += --target=generic-gnu
-ADDITIONAL_CPPFLAGS = -std=c++0x
 ADDITIONAL_CXXFLAGS = -std=c++0x
 endif
 
 ifeq ($(findstring $(ARCH),$(ARMv7_ARCHS) $(ARMv7L_ARCHS)),$(ARCH))
 CONFIGURE_ARGS += --target=armv7-linux-gcc
-CONFIGURE_ARGS += --disable-neon
```

Comment:
Disabling of neon was done in this PR https://github.com/SynoCommunity/spksrc/pull/5620
This was the comit: https://github.com/SynoCommunity/spksrc/pull/5620/commits/6ad7f5f927d86a7c56c5c7402614d33e3d642545

So from the commit log it states disabling neon (i.e. which includes `as` optimizations) on armv7 when using a compiler <= 4.9, basically on dsm6.  I suggest the following change:
```
include ../../mk/spksrc.common.mk

ifeq ($(findstring $(ARCH),$(ARMv7_ARCHS) $(ARMv7L_ARCHS)),$(ARCH))
CONFIGURE_ARGS += --target=armv7-linux-gcc
ifeq ($(call version_lt, ${TC_GCC}, 5),1)
CONFIGURE_ARGS += --disable-neon
endif
endif

```